zap/src/main/java/org/zaproxy/zap/extension/ascan/ActiveScan.java

```diff
@@ -222,7 +225,8 @@ public void hostNewScan(int id, String hostAndPort, HostProcess hostThread) {}
     @Override
     public void hostProgress(int id, String hostAndPort, String msg, int percentage) {
         // Calculate the percentage based on the average of all of the host processes
-        // This is an approximation as different host process make significantly different times
+        // This is an approximation as different host process make significantly
```

Comment:
Can be reverted.

Reply:
done automatically by spotlessapply.

Reply:
Most likely because of a change done/reverted in the meantime, so this should be reverted as well.

Reply:
oh ... but in case if modified in future, spotless will again do the same.

Reply:
You sure? I just reverted the change and `spotlessApply` didn't reformat it.

Reply:
oh .. but it does. let me try again